### PR TITLE
Feature/evidence note submit

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**
Submit Evidence note
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_sprints/taskboard/EA%20-%20WEEE%20Team/EA%20-%20WEEE/Phase%203/Beta%20Sprint%202?workitem=42739

The feature needed some of the changes from the code that is on Guidos View Aatf Draft story branch so ignore those changes that are to retrieve the Evidence note from the database, integration and unit tests will be done as part of that.

### CODE REVIEW

File: src/EA.Weee.RequestHandlers/AatfEvidence/GetEvidenceNoteHandler.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+        public GetEvidenceNoteHandler(IWeeeAuthorization authorization,
+            IGenericDataAccess genericDataAccess,
+            IAatfDataAccess aatfDataAccess, 
+            ISchemeDataAccess schemeDataAccess,
+            IEvidenceDataAccess evidenceDataAccess,
+            IMapper mapper)
+        {
+            this.authorization = authorization;
+            this.genericDataAccess = genericDataAccess;
```
[the remark this answers is not in the document]
Comment (JohnPadgettCivica) [reply src/EA.Weee.RequestHandlers/AatfEvidence/GetEvidenceNoteHandler.cs:34]: Will update this as part of the view story.

File: src/EA.Weee.RequestHandlers/AatfEvidence/GetEvidenceNoteHandler.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            this.aatfDataAccess = aatfDataAccess;
```
[the remark this answers is not in the document]
Comment (JohnPadgettCivica) [reply src/EA.Weee.RequestHandlers/AatfEvidence/GetEvidenceNoteHandler.cs:35]: Will update this as part of the view story.

File: src/EA.Weee.RequestHandlers/AatfEvidence/GetEvidenceNoteHandler.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (meglenapnovoselska) [reply src/EA.Weee.RequestHandlers/AatfEvidence/GetEvidenceNoteHandler.cs:34]: okay, leave it there

File: src/EA.Weee.Web/Areas/Aatf/Mappings/ToViewModel/ViewEvidenceNoteViewModelMap.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    public class ViewEvidenceNoteViewModelMap : IMap<ViewEvidenceNoteMapTransfer, ViewEvidenceNoteViewModel>
+    {
+        public ViewEvidenceNoteViewModel Map(ViewEvidenceNoteMapTransfer source)
+        {
+            Guard.ArgumentNotNull(() => source, source);
+
+            //var singleAatf = source.Aatfs.Where(a => a.FacilityType.Equals(FacilityType.Aatf) && ((int)a.ComplianceYear).Equals(SystemTime.Now.Year));
+
+            var model = new ViewEvidenceNoteViewModel
```
[the remark this answers is not in the document]
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Aatf/Mappings/ToViewModel/ViewEvidenceNoteViewModelMap.cs:15]: will update this one in the view branch.

**Review by meglenapnovoselska — APPROVED** [review]

**Pull request merged**